package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "max7219-display",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
This actually gets done automatically by `npm version` when publishing to ensure there is always a matching git tag for a release.

I'd also say that the sort of changes going on here are new APIs and functionality, so this would be a minor release (2.1.0) not a patch. I'll update accordingly.